modules/ROOT/pages/anypoint-mq/3.x/anypoint-mq-connector-reference.adoc

```diff
@@ -0,0 +1,526 @@
+[%header%autowidth.spread]
+|===
+| Field | Type | Description | Default Value | Required
+| Destination a| String | The destination the message belongs to. |  | x
+| Headers a| Object | The headers of the message. |  | x
+| Properties a| Object | The properties of the message. |  | x
+| Message Id a| String | The ID of the message. |  | x
+| Delivery Count a| Number | The number of times the message was delivered. |  | x
+| Content Type a| String | The content type of the message body |  |
+| Ack Token a| String | The acknowledgment token of the message, passed to Ack/Nack operations to perform ack/nack manually.
+ This token will be present only when MANUAL ACK mode is used. |  |
+|===
+
+[[circuitBreaker]]
+=== Circuit Breaker
+
+[%header%autowidth.spread]
+|===
+| Field | Type | Description | Default Value | Required
+| On Error Types a| String | The error types whose occurrence during the flow execution will count as a failure in the circuit.
+ Error counts only when the flow finishes with an error propagation.
+ By default all the errors will count as a failure. |  |
+| Errors Threshold a| Number | How many errors, of those considered by this circuit, have to occur for the circuit to be opened. |  | x
+| Trip Timeout a| Number | How long will the circuit remain open once the errorsThreshold is reached |  | x
+| Trip Timeout Unit a| Enumeration, one of:
+
+** NANOSECONDS
+** MICROSECONDS
+** MILLISECONDS
+** SECONDS
+** MINUTES
+** HOURS
+** DAYS | The time unit bound to the tripTimeout | MILLISECONDS |
+|===
+
+[[repeatable-in-memory-stream]]
+=== Repeatable In Memory Stream
+
+[%header%autowidth.spread]
+|===
+| Field | Type | Description | Default Value | Required
+| Initial Buffer Size a| Number | The amount of memory that will be allocated to consume the stream and provide random access to it. If the stream contains more data than can be fit into this buffer, then the buffer expands according to the bufferSizeIncrement attribute, with an upper limit of maxInMemorySize. |  |
+| Buffer Size Increment a| Number | This is by how much the buffer size expands if it exceeds its initial size. Setting a value of zero or lower means that the buffer should not expand, meaning that a STREAM_MAXIMUM_SIZE_EXCEEDED error is raised when the buffer gets full. |  |
+| Max Buffer Size a| Number | The maximum amount of memory to use. If more than that is used then a STREAM_MAXIMUM_SIZE_EXCEEDED error is raised. A value lower than or equal to zero means no limit. |  |
+| Buffer Unit a| Enumeration, one of:
+
+** BYTE
+** KB
+** MB
+** GB | The unit in which all these attributes are expressed |  |
+|===
+
+[[repeatable-file-store-stream]]
+=== Repeatable File Store Stream
+
+[%header%autowidth.spread]
+|===
+| Field | Type | Description | Default Value | Required
+| Max In Memory Size a| Number | Defines the maximum memory that the stream should use to keep data in memory. If more than that is consumed then it will start to buffer the content on disk. |  |
```

Comment:
The maximum amount of memory that the stream can hold locally before content is buffered to disk.